lib/kube/proxy/remotecommand.go

```diff
@@ -116,6 +116,9 @@ func upgradeRequestToRemoteCommandProxy(req remoteCommandRequest, exec func(*rem
 		go proxy.resizeQueue.handleResizeEvents(proxy.resizeStream)
 	}
 	err = exec(proxy)
+	if !isRelevantWebsocketError(err) {
+		err = nil
+	}
```

Comment:
I'm a little confused here.

If the if statement does not execute, and `err` is not nilled out, how are we supposed to handle the error? We immediately overwrite it, which doesn't seem correct.

Reply:
We overwrite it only after sending it down the line. The comment below mentioned that we return nil as it's a hijacked connection, so we handle the error by sending status.